### PR TITLE
test(api): added postman verbs data tests

### PR DESCRIPTION
**Pull request by asimmohx**
This PR creates the setup for the following tasks:

https://dev.azure.com/rbheBoards/OpenAMT/_workitems/edit/7422/?workitem=11843
https://dev.azure.com/rbheBoards/OpenAMT/_workitems/edit/7420/?workitem=11845
https://dev.azure.com/rbheBoards/OpenAMT/_workitems/edit/7415/?workitem=11841
https://dev.azure.com/rbheBoards/OpenAMT/_workitems/edit/7415/?workitem=11839

### CODE REVIEW

File: src/test/collections/data/mps_verbdata.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "_TEST_CASE_6597": "MPS: Rest API-  http tampering- using PUT method for mps api's instead of POST",
+    "url": "http://{{host}}/api/v1/authorize",
```
Comment (rsdmike) [inline src/test/collections/data/mps_verbdata.json:4]: `http://` should be using the `{{procotol}}` env variable

**Review by rsdmike — CHANGES_REQUESTED** [review]
don't love verbdata.json as a file name. I think something like `security_api_test.data.json` and `using-verbs` should follow the `security_api_test.postman_collections,json`

File: src/test/collections/data/mps_verbdata.json (hunk 1 above)
Comment (asimmohx) [reply src/test/collections/data/mps_verbdata.json:4]: oh I missed this earlier. Let me use update that.

Comment (asimmohx) [reply src/test/collections/data/mps_verbdata.json]: Renamed files as well

**Pull request merged**